miasm/core/cpu.py

```diff
@@ -1085,7 +1087,10 @@ def guess_mnemo(cls, bs, attrib, pre_dis_info, offset):
             (l, fmask, fbits, fname, flen), vals = branch
 
             if flen is not None:
-                l = flen(attrib, fname_values)
+                try:
+                    l = flen(attrib, fname_values, bs, offset_b)
```

Comment:
With this new code, flen can raise IOError as the user can get bits from the bs.
You must have the same behavior as getbits below in case of IOError.

Reply:
I fixed it in the latest commit.